main_app/static/imgs/plants/general-use/home-image.png:Zone.Identifier

```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]

```

Comment:
remove the zone.identifier for the images or add **/*Zone.Identifier to .gitignore

Reply:
Removed